installer.cfg

```diff
@@ -13,18 +13,24 @@ version=3.8.5
 # These must have wheels on PyPI:
 # https://pynsist.readthedocs.io/en/latest/cfgfile.html#include-section
 # Look them up here: https://pypi.org/
+
 pypi_wheels = pynput==1.6.8
     win10toast==0.9
     six==1.15.0
     setuptools==49.6.0
     pywin32==228
     PyQt5==5.15.0
-
-files=stats.py
+    sip==5.4.0
+    PyQt5-sip==12.8.1
+    pandas==1.1.1
+    numpy==1.19.1
+    pytz==2020.1
+    python-dateutil==2.8.1
 
 # Other files and folders that should be installed
 # files = LICENSE
 #     data_files
+# files=keylogger/
```

Comment:
Why is this commented out? If pynsist automatically pulls the whole module, then we can delete this?

Reply:
Yeah, we could delete it, originally I was trying to get it work with files but looks like we don't need those. :)